coreneuron/io/global_vars.cpp

```diff
@@ -100,7 +101,7 @@ void set_globals(const char* path, bool cli_global_seed, int cli_global_seed_val
                 }
                 it = n2v->find(name);
                 if (it != n2v->end()) {
-                    nrn_assert(it->second.first == (size_t)n);
+                    nrn_assert(it->second.first == (size_t) n);
```

Comment:
I kind of like to have casts glued to what we are casting from, but this is a matter of personal taste.

Reply:
I am ok with such change. Is there such option in clang-format? If so we can update it.

Reply:
`SpaceAfterCStyleCast: false`

Reply:
Let's not discuss style here - since the formatting comes from hpc coding conventions this discussion could be discussed there. We certainly wouldn't want to divert from the format file there, right?